eZ/Publish/Core/Persistence/Legacy/Tests/Content/FieldValue/Converter/ISBNTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * File containing the ISBNTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Persistence\Legacy\Tests\Content\FieldValue\Converter;
+
+use eZ\Publish\Core\FieldType\FieldSettings;
+use eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter\ISBNConverter;
+use eZ\Publish\Core\Persistence\Legacy\Content\StorageFieldDefinition;
+use eZ\Publish\SPI\Persistence\Content\Type\FieldDefinition as PersistenceFieldDefinition;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * Test for ISBNConverter in Legacy Storage.
+ */
+class ISBNTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var \eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter\ISBNConverter
+     */
+    protected $converter;
+
+    protected function setUp()
+    {
+        $this->converter = new ISBNConverter();
+    }
+
+    /**
+     * @dataProvider data_toFieldDefinition
+     *
+     * @covers \eZ\Publish\Core\Persistence\Legacy\Content\FieldValue\Converter\ISBNConverter::toFieldDefinition
+     */
+    public function testToFieldDefinition($dataInt, $excpectedIsbn13Value)
+    {
+        $fieldDef = new PersistenceFieldDefinition();
+        $storageDefinition = new StorageFieldDefinition([
+            'dataInt1' => $dataInt,
+        ]);
+
+        $this->converter->toFieldDefinition($storageDefinition, $fieldDef);
+
+        /** @var FieldSettings $fieldSettings */
+        $fieldSettings = $fieldDef->fieldTypeConstraints->fieldSettings;
+        self::assertSame($excpectedIsbn13Value, $fieldSettings['isISBN13']);
+    }
+
+    public function data_toFieldDefinition()
```

Comment:
Sorry, didn't notice this before, PSR-2 method names must be in `camelCase`. 
I would use very explicit name such as `providerForTestToFieldDefinition` but might be shorter as long as there are no underscores.
Unfortunately php-cs-fixer won't protect us in these cases :(